### ISSUE TITLE
Type conversion for hex numbers fails

### ISSUE THREAD

**Issue by maxpietsch** [MRtrix3/mrtrix3#376]
The [to](http://jdtournier.github.io/mrtrix3-dev-doc/namespaceMR.html#ad5a737bfbb3fcb51accf1a329da6fb7c) function that is used in `load_matrix` which in turn is used in `load_transform` does not convert properly from hexadecimal floating point numbers to float. Flirt produces transformation files that look like this:
~~~~~
0x1.07ef6e95b728ep+0  0x1.031c0c475b32p-5  -0x1.3899e4bc371b2p-7  -0x1.e083b41b808ecp+2  
-0x1.d1dced01aed46p-6  0x1.0c6710bc82722p+0  0x1.9e40a37ec5554p-7  -0x1.87e1e1cb749f1p+1  
-0x1.a6a6e40ed4456p-7  -0x1.a594d6bcbf3edp-8  0x1.073bb88d68f26p+0  0x1.7fe79ddb71252p+1  
0x0p+0  0x0p+0  0x0p+0  0x1p+0 
~~~~~
which is parsed as signed zeros. I figure best would be to check for hex numbers in the `to` function but for now I added warnings to `transformcalc` 3a03dbbebe8f3b81a.

**Comment by jdtournier** [comment]
Actually, that a bug in FSL... Fixed in the new version (5.0.9), released just a week or two ago. I don't think it's worth investing any time trying to support that format...

**Comment by maxpietsch** [comment]
Ok, I'll leave the warning in tranformcalc for the time being and close this.
